boot/bootutil/src/bootutil_priv.h

```diff
@@ -129,18 +129,21 @@ struct boot_swap_state {
 #endif
 
 /** Number of image slots in flash; currently limited to two. */
-#define BOOT_NUM_SLOTS             2
+#define BOOT_NUM_SLOTS                  2
 
 /** Maximum number of image sectors supported by the bootloader. */
-#define BOOT_STATUS_STATE_COUNT    3
-#define BOOT_STATUS_MAX_ENTRIES    BOOT_MAX_IMG_SECTORS
+#define BOOT_STATUS_STATE_COUNT         3
+#define BOOT_STATUS_MAX_ENTRIES         BOOT_MAX_IMG_SECTORS
 
-#define BOOT_STATUS_SOURCE_NONE    0
-#define BOOT_STATUS_SOURCE_SCRATCH 1
-#define BOOT_STATUS_SOURCE_SLOT0   2
+#define BOOT_PRIMARY_SLOT               0
+#define BOOT_SECONDARY_SLOT             1
 
-#define BOOT_FLAG_IMAGE_OK         0
-#define BOOT_FLAG_COPY_DONE        1
+#define BOOT_STATUS_SOURCE_NONE         0
+#define BOOT_STATUS_SOURCE_SCRATCH      1
+#define BOOT_STATUS_SOURCE_PRIMARY_SLOT 2
+
+#define BOOT_FLAG_IMAGE_OK              0
+#define BOOT_FLAG_COPY_DONE             1
 
```

Comment:
To pass Mynewt CI, please add the following at this line:
```
#if defined(MCUBOOT_MYNEWT)                                                     
    #define FLASH_AREA_IMAGE_PRIMARY   FLASH_AREA_IMAGE_0                           
    #define FLASH_AREA_IMAGE_SECONDARY FLASH_AREA_IMAGE_1                           
#endif
```
`newt` auto-generates the `FLASH_AREA_IMAGE_[0|1]` symbols from `bsp.yml` and this adds a compatibility definition.